src/Components/ReviewScore/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReviewStandard } from '@/Types/review';
+import styled from 'styled-components';
+import { StarScore } from './StarScore';
+
+interface ReviewScoreProps {
+  /** 리뷰 평가 기준 */
+  standard: ReviewStandard;
+  /** 평가 점수 */
+  score: number;
+}
+
```

Comment:
type을 내보내주면 다른 페이지에서도 재사용하기 쉬워질 것 같아요!

```suggestion

export interface ReviewScoreProps {
  /** 리뷰 평가 기준 */
  standard: ReviewStandard;
  /** 평가 점수 */
  score: number;
}

```